templates/repo/actions/runs_list.tmpl

```diff
@@ -6,38 +6,42 @@
 	</div>
 	{{end}}
 	{{range .Runs}}
-		<div class="flex-item tw-items-center">
-			<div class="flex-item-leading">
-				{{template "repo/actions/status" (dict "status" .Status.String)}}
-			</div>
-			<div class="flex-item-main">
-				<a class="flex-item-title" title="{{.Title}}" href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
-					{{if .Title}}{{.Title}}{{else}}{{ctx.Locale.Tr "actions.runs.empty_commit_message"}}{{end}}
-				</a>
-				<div class="flex-item-body">
-					<span><b>{{if not $.CurWorkflow}}{{.WorkflowID}} {{end}}#{{.Index}}</b>:</span>
-					{{- if .ScheduleID -}}
-						{{ctx.Locale.Tr "actions.runs.scheduled"}}
-					{{- else -}}
-						{{ctx.Locale.Tr "actions.runs.commit"}}
-						<a href="{{$.RepoLink}}/commit/{{.CommitSHA}}">{{ShortSha .CommitSHA}}</a>
-						{{ctx.Locale.Tr "actions.runs.pushed_by"}}
-						<a href="{{.TriggerUser.HomeLink}}">{{.TriggerUser.GetDisplayName}}</a>
-					{{- end -}}
+		{{$CurWorkflow := $.CurWorkflow}}
+		{{$WorkflowName := .Workflow.Name}}
+		{{with .ActionRun}}
```

Comment:
The variable shadowing is more complex now.  And the name `Run` also seems ambiguous

I think it's better to do so:

```
{{range $run = .WorkflowActionRuns}}
    {{$actionRun := $run.ActionRun}}
    {{$workflow = $run.Workflow}}
    {{$actionRun.TItle}}
    ...
```

Reply:
Thanks for the idea, I did the changes.